src/pages/index.js

```diff
@@ -5,31 +5,82 @@ import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import "./index.css";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
+
+/*------------------------------------------------------------------*/
+/*                             API                                  */
+/*------------------------------------------------------------------*/
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5764b9be-6485-485e-a9f1-d6c0fb47f479",
+    "Content-Type": "application/json",
+  },
+});
+
+/*------------------------------------------------------------------*/
+/*                          USER INFO                               */
+/*------------------------------------------------------------------*/
+
+const profileUserInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  descriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+
+profileUserInfo.getUserInfo();
+
+/*------------------------------------------------------------------*/
+/*                          USER INFO                               */
+/*------------------------------------------------------------------*/
 
 const editFormValidator = new FormValidator(
   Constants.validationSettings,
   Constants.profileEditForm
 );
+
+editFormValidator.enableValidation();
+
 const addFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```